games-emulation/ryujinx/ryujinx-1.2.81.ebuild

```diff
@@ -0,0 +1,361 @@
+system.memory@4.5.4
+system.memory@4.5.5
+system.numerics.vectors@4.3.0
+system.numerics.vectors@4.4.0
+system.numerics.vectors@4.5.0
+system.objectmodel@4.3.0
+system.private.uri@4.3.0
+system.reactive@6.0.1
+system.reflection.emit.ilgeneration@4.3.0
+system.reflection.emit.lightweight@4.3.0
+system.reflection.emit@4.3.0
+system.reflection.extensions@4.3.0
+system.reflection.metadata@1.6.0
+system.reflection.metadata@5.0.0
+system.reflection.metadata@8.0.0
+system.reflection.primitives@4.3.0
+system.reflection.typeextensions@4.3.0
+system.reflection@4.3.0
+system.resources.resourcemanager@4.3.0
+system.runtime.compilerservices.unsafe@4.5.2
+system.runtime.compilerservices.unsafe@4.5.3
+system.runtime.compilerservices.unsafe@4.7.1
+system.runtime.compilerservices.unsafe@5.0.0
+system.runtime.compilerservices.unsafe@6.0.0
+system.runtime.extensions@4.3.0
+system.runtime.interopservices@4.3.0
+system.runtime@4.3.0
+system.security.accesscontrol@4.5.0
+system.security.principal.windows@4.5.0
+system.text.encoding.codepages@4.5.1
+system.text.encoding.codepages@8.0.0
+system.text.encoding@4.3.0
+system.text.encodings.web@8.0.0
+system.text.json@8.0.0
+system.text.json@8.0.3
+system.threading.tasks.extensions@4.5.4
+system.threading.tasks@4.3.0
+system.threading@4.3.0
+tmds.dbus.protocol@0.15.0
+unicornengine.unicorn@2.0.2-rc1-fb78016
+"
+
+inherit check-reqs desktop dotnet-pkg xdg
+
+DESCRIPTION="Experimental Nintendo Switch Emulator written in C#"
+HOMEPAGE="https://ryujinx-emulator.com/
+	https://github.com/Ryubing/Ryujinx"
+
+if [[ "${PV}" == *9999* ]] ; then
+	inherit git-r3
+
+	EGIT_REPO_URI="https://github.com/Ryubing/${PN^}.git"
+else
+	SRC_URI="https://github.com/Ryubing/${PN^}/archive/refs/tags/${PV}.tar.gz
+		-> ${P}.tar.gz"
+	S="${WORKDIR}/${P^}"
+
+	KEYWORDS="~amd64"
+fi
+
+SRC_URI+=" ${NUGET_URIS} "
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="
+	app-arch/brotli
+	dev-libs/expat
+	dev-libs/icu
+	dev-libs/libxml2
+	dev-libs/openssl
+	dev-libs/wayland
+	media-gfx/graphite2
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/harfbuzz
+	media-libs/libglvnd[X]
+	media-libs/libpng
+	media-libs/libpulse
+	media-libs/libsdl2
+	media-video/pipewire
+	x11-libs/gtk+:3
+	x11-libs/libX11
+"
+
+CHECKREQS_DISK_BUILD="3G"
+DOTNET_PKG_PROJECTS=(
+	"src/${PN^}/${PN^}.csproj"
+)
+PATCHES=(
+	"${FILESDIR}/${P}-better-defaults.patch"
+)
+
+DOCS=( README.md distribution/legal/THIRDPARTY.md )
+
+DOTNET_PKG_BUILD_EXTRA_ARGS=( -p:ExtraDefineConstants=DISABLE_UPDATER )
+
+pkg_setup() {
+	check-reqs_pkg_setup
+	dotnet-pkg_pkg_setup
+}
+
+src_unpack() {
+	dotnet-pkg_src_unpack
+
+	if [[ -n "${EGIT_REPO_URI}" ]] ; then
+		git-r3_src_unpack
+	fi
+}
+
+src_prepare() {
+	sed "s|1.0.0-dirty|${PV}|g" -i src/*/*.csproj || die
+
+	# The patch below can be removed once >=dotnet-sdk-8.0.401 is generally available.
+	sed 's|Include="Microsoft.CodeAnalysis.CSharp" Version="4.9.2"|Include="Microsoft.CodeAnalysis.CSharp" Version="4.8.0"|' \
```

Comment:
According to the preceding comment this patch can be removed.

Reply:
I removed sed, thanks!